src/pages/dashboard/.Map.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState, useMemo, useEffect } from 'react'
+
+import { json } from 'd3-fetch'
+import { nest } from 'd3-collection'
+import { sum } from 'd3-array'
+
+import _Map from './charts/_Map'
+import { path } from './charts/_Map'
```

Comment:
Nitpick, but I've usually seen this done as `import _Map, { path } from './charts/_Map'` - there an advantage either way?

Reply:
no, i actually prefer your way, just forgot about it this time